exercises/practice/perfect-numbers/.meta/config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "authors": ["meatball133"],
```

Comment:
Just a little FYI. This formatting is not the one used by `configlet fmt`. That is totally fine, but if you ever want to enable `configlet fmt` linting, this is something to consider.